update/renewCerts.bash

```diff
@@ -6,11 +6,29 @@ cd $scriptDir
 source ../conf
 
 letsencFolder=/etc/letsencrypt
+dateNow=$(date -d "now" +%s)
+
 
 # Update all certs with cli.ini file in folder for Let's Encrypt
 for configFile in $(ls $letsencFolder/*.cli.ini); do
-	echo "Renewing cert using $configFile"
-	../letsencrypt/letsencrypt-auto certonly --renew-by-default --config $configFile
+	
+	primaryDomain=`grep "^\s*domains" $configFile | sed "s/^\s*domains\s*=\s*//" | sed 's/(\s*)\|,.*$//'`
```

Comment:
This will only match the first domain in the list (which is necessary for this script to work).